data-source/jsonnet/england-wales/blocks/individual/personal-details/sex.jsonnet

```diff
@@ -69,7 +69,7 @@ local guidance = {
       goto: {
         block: 'marriage-type',
         when: [
```

Comment:
Do the rules on the question variants shown here also need updating?

Reply:
Hadn't spotted these, but will have to double check because the guidance shown isn't related to skipping the marriage questions.

Reply:
Checked with Charlotte and have now updated the question variants.